unit_test/main_tests/main_platform_test.py

```diff
@@ -123,12 +123,10 @@ def test_archs_platform_native(platform, intercepted_build_args, monkeypatch):
     main()
     build_options = intercepted_build_args.args[0]
 
-    if platform == 'linux':
+    if platform in {'linux', 'macos'}:
```

Comment:
We might want to think about this, since we're used seeing (and checking for!) the 3-part if of "platform is Linux/mac/Windows"?

Then again, now it's Unix vs. Windows, so that also makes kind of sense?

Reply:
In the regular code, I use completeness checking to ensure we have all three, and (I am pretty sure) MyPy can't do completeness checking if you use a set (at least, currently). But here, we aren't checking as carefully with MyPy, and this combines the similar platforms, keeping the tests a little simpler.

Reply:
Sure, didn't notice this was in the tests, actually. So then it's definitely OK :-)